app/helpers/maintenance_tasks/tasks_helper.rb

```diff
@@ -101,13 +101,39 @@ def csv_file_download_path(run)
       )
     end
 
+    # Returns the appropriate value for a Proc, lambda, symbol, or other types.
+    #
+    # @param validator [ActiveModel::Validations::*] The validator object used to resolve the value.
```

Comment:
Yep - it would always be the inclusion validator, I refactored the helper method and it's arguments in https://github.com/Shopify/maintenance_tasks/pull/962/commits/4eec6124987b914fd8a8c3c87e6703475944b001